web/yo/app/scripts/controllers/gene.js

```diff
@@ -1637,7 +1634,7 @@ angular.module('oncokbApp')
             }
 
             $rootScope.reviewMode = false;
-            $scope.displayCheck = function(uuid, reviewObj, mutationReview, tumorReview, treatmentReview) {
+            $scope.displayCheck = function(uuid, reviewObj, mutationReview, tumorReview, treatmentReview, precise) {
```

Comment:
That's a good suggestion. I just added default setting for precise. For the rest, I am not sure if we need to. Since inside the function, we always check if it is null or not before using it.